easybuild/easyconfigs/e/EMAN2/EMAN2-2.3_fix_bad_use_cx11_abi_setting.patch

```diff
@@ -0,0 +1,15 @@
+Don't use -D_GLIBCXX_USE_CXX11_ABI=0, the code will be broken.
```

Comment:
@akesandgren Maybe you should elaborate a bit, and mention how things are broken, i.e. which errors are hit? That'll make it a lot easier to find this again if we'd run into the same problem again later...

Also, was this reported upstream? If so, please include a reference to the issue

```
Don't use -D_GLIBCXX_USE_CXX11_ABI=0, the code will be broken:

$ python -c 'import EMAN2'
...
TypeError: No registered converter was able to produce a C++ rvalue of type std::string from this Python object of type str
```

Reply:
No, not reported upstream. They insist on only building with conda, and i suspect they are using the same flag for their boost-python setup.

Reply:
🤦‍♂ 
OK, fine